app/assets/javascripts/src/addressLookup.js

```diff
@@ -28,8 +28,19 @@
         }
 
         function showLookupError() {
-            $('#postcodeSearchGroup').find('.error-message').remove();
-            $('#postcodeSearch').before('<p class="error-message">' + messages.errors.postcodeLookupError + '</p>').closest('.form-group').addClass('error');
+            var isContains = $('#postcodeSearchOnly').text().indexOf('Enter a valid postcode') > -1;
```

Comment:
this is a copy of Zahir's which I have commented on in his PR. I think in this case it would have been better to wait for Zahir's PR before raising this

Reply:
just make sure you keep yours inline with Zahir's changes

Reply:
have updated this now to match Zahirs 👍🏻